rtl.roc

```diff
@@ -13,19 +14,19 @@ import cli.Utc exposing [Utc]
 import Parser
 import CodeGen
 import cli.Arg
-import cli.Arg.Opt as Opt
-import cli.Arg.Cli as Cli
+import cli.Arg.Opt
+import cli.Arg.Cli
 
 main =
     start = Utc.now!
 
     cliParser =
-        Cli.build {
-            maybeInputDir: <- Opt.maybeStr { short: "i", long: "input-directory", help: "The directory containing the templates to be compiled. Defaults to the current directory." },
-            maybeOutputDir: <- Opt.maybeStr { short: "o", long: "output-directory", help: "The directory Pages.roc will be written to. Defaults to the current directory." },
-            maybeExtension: <- Opt.maybeStr { short: "e", long: "extension", help: "The extension of the template files the CLI will search for. Defaults to `rtl`." },
+        { Arg.Cli.combine <-
```

Comment:
Can this (and the others) be `Cli.combine` instead of `Arg.Cli.combine`?

Reply:
You prefer the alias? no problem.

Reply:
I do, but on second thought it doesn't really matter either way.